src/logic/azext_logic/manual/custom.py

```diff
@@ -35,3 +69,114 @@ def logic_workflow_update(cmd,
+        'tags': tags,
+        'sku': {'name': sku},
+        'integration_service_environment': integration_service_environment,
+        'state': state if state else 'Enabled',
+    }
+    return client.create_or_update(resource_group_name=resource_group_name,
+                                   integration_account_name=name,
+                                   integration_account=integration_account)
+
+
+def logic_integration_account_import(cmd, client,
+                                     resource_group_name,
+                                     name,
+                                     input_path,
+                                     location=None,
+                                     tags=None,
+                                     sku=None, ):
+    if 'properties' not in input_path:
+        raise CLIError(str(input_path) +
+                       " does not contain a 'properties' key")
+
+    integration_service_environment = input_path['properties'].get(
+        'integrationServiceEnvironment', None)
+    integration_account = {
+        'location': input_path.get('location', location),
+        'tags': input_path.get('tags', tags),
+        'sku': input_path.get('sku', {'name': sku}),
+        'integration_service_environment': integration_service_environment,
+        'state': input_path['properties'].get('state', 'Enabled'),
+    }
+    return client.create_or_update(resource_group_name=resource_group_name,
+                                   integration_account_name=name,
+                                   integration_account=integration_account)
+
+
+def logic_integration_account_update(cmd, client,
+                                     name,
+                                     resource_group_name,
+                                     tags=None,
+                                     sku=None,
+                                     integration_service_environment=None,
+                                     state=None):
+    import json
+    if isinstance(integration_service_environment, str):
+        integration_service_environment = json.loads(
+            integration_service_environment)
+    integration_account = {
+        'location': None,
+        'tags': tags,
+        'sku': {'name': sku},
+        'integration_service_environment': integration_service_environment,
+        'state': state,
+    }
+    return client.update(resource_group_name=resource_group_name,
+                         integration_account_name=name,
+                         integration_account=integration_account)
+
+
+class MapCreate(_MapCreate):
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        from azure.cli.core.aaz import AAZFileArg
+        args_schema = super()._build_arguments_schema(*args, **kwargs)
+        args_schema.map_content = AAZFileArg(
+            options=["--map-content"],
+            help="The content.",
+        )
+        args_schema.content._registered = False
```

Comment:
The conent argument by default can support file input like json and yaml files, so it doesn't to add another argument.